rskj-core/src/test/java/co/rsk/rpc/modules/eth/EthModuleTest.java

```diff
@@ -52,10 +53,46 @@ public void callSmokeTest() {
                 .thenReturn(blockResult);
         when(blockResult.getBlock()).thenReturn(block);
 
-        byte[] hreturn = TypeConverter.stringToByteArray("hello");
+        byte[] hReturn = TypeConverter.stringToByteArray("hello");
+        ProgramResult executorResult = mock(ProgramResult.class);
+        when(executorResult.getHReturn())
+                .thenReturn(hReturn);
+
+        ReversibleTransactionExecutor executor = mock(ReversibleTransactionExecutor.class);
+        when(executor.executeTransaction(eq(blockResult.getBlock()), any(), any(), any(), any(), any(), any(), any()))
+                .thenReturn(executorResult);
+
+        EthModule eth = new EthModule(
+                null,
+                anyByte(),
+                null,
+                null,
+                executor,
+                retriever,
+                null,
+                null,
+                null,
+                new BridgeSupportFactory(
+                        null, null, null));
+
+        String result = eth.call(args, "latest");
+        assertThat(result, is(TypeConverter.toUnformattedJsonHex(hReturn)));
```

Comment:
Same here, to be clear whats the expected value.

Reply:
done